### PR TITLE
adressing add pylint workflow and fix all issues 

### PR DESCRIPTION
**Pull request by sigmadawg**
addresses #262

### CODE REVIEW

File: .github/workflows/pylint.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
```
Comment (djeck1432) [inline .github/workflows/pylint.yml:13]: we use `python3.11`

File: apps/data_handler/celery_app/tasks.py [hunk 2]
```diff
@@ -1,78 +1,35 @@
...
-
 from data_handler.db.crud import DBConnector
 from data_handler.db.models import OrderBookModel
-
 from .celery_conf import app
 
 connector = DBConnector()
 
 
-# @app.task(name="run_loan_states_computation_for_hashtack_v0")
```
Comment (djeck1432) [inline apps/data_handler/celery_app/tasks.py:29]: please, don't remove any code

File: apps/data_handler/celery_app/tasks.py [hunk 3]
```diff
@@ -1,78 +1,35 @@
...
-# from data_handler.handlers.loan_states.nostra_mainnet.run import NostraMainnetStateComputation
 from data_handler.handlers.liquidable_debt.protocols import (
     hashstack_v0,
     hashstack_v1,
     nostra_alpha,
     nostra_mainnet,
     zklend,
 )
-
-# from data_handler.handlers.loan_states.hashtack_v0.run import HashtackV0StateComputation
```
Comment (djeck1432) [inline apps/data_handler/celery_app/tasks.py:13]: please, leave these imports

File: apps/data_handler/celery_app/tasks.py [hunk 4]
```diff
@@ -1,78 +1,35 @@
...
-#     computation.run()
-#
-#     logging.info(
-#         "Finished Hashtack V0 loan state computation, Time taken: %s seconds",
-#         monotonic() - start,
-#     )
-
-
-# @app.task(name="run_loan_states_computation_for_hashtack_v1")
-# def run_loan_states_computation_for_hashtack_v1():
```
Comment (djeck1432) [inline apps/data_handler/celery_app/tasks.py:45]: don't remove any code

File: apps/data_handler/celery_app/tasks.py [hunk 5]
```diff
@@ -86,26 +43,15 @@ def run_loan_states_computation_for_nostra_alpha():
     )
 
 
-# @app.task(name="run_loan_states_computation_for_nostra_mainnet")
```
Comment (djeck1432) [inline apps/data_handler/celery_app/tasks.py:89]: don't remove any code

File: apps/data_handler/db/models/loan_states.py [hunk 6]
```diff
@@ -1,3 +1,24 @@
...
+
+
+class LoanState(BaseState):
+    """
+    Model for the `loan_state` table, tracking protocol-specific loan details.
+    """
+
+    __
+
+
```
Comment (djeck1432) [inline apps/data_handler/db/models/loan_states.py:21]: you have duplicated code

File: apps/data_handler/handler_tools/constants.py [hunk 7]
```diff
@@ -114,8 +123,10 @@ class ProtocolAddresses:
 NOSTRA_EVENTS_MAPPING = {
     "Mint": "process_debt_mint_event",
     "Burn": "process_debt_burn_event",
-    "nostra::core::tokenization::lib::nostra_token::NostraTokenComponent::Burn": "process_debt_burn_event",
```
Comment (djeck1432) [inline apps/data_handler/handler_tools/constants.py:117]: why did you remove this code?

File: apps/data_handler/handler_tools/nostra_alpha_settings.py [hunk 8]
```diff
@@ -27,29 +33,19 @@
 
 # Source: https://docs.nostra.finance/lend/deployed-contracts/lend-alpha#asset-contracts.
 NOSTRA_ALPHA_TOKEN_ADDRESSES: list[str] = [
-    # '0x0061d892cccf43daf73407194da9f0ea6dbece950bb24c50be2356444313a707',  # iWBTC
```
Comment (djeck1432) [inline apps/data_handler/handler_tools/nostra_alpha_settings.py:30]: Don't remove any code, please!!!!

**Review by djeck1432 — CHANGES_REQUESTED** [review]
Please, sync with master and don't remove any code

**Comment by sigmadawg** [comment]
sorry for the inconvenience, im a complete newbie, i tried linting the codes using black, but that didnt entirely fix all the long line issues, so i did some manually and messed up a lot.
If you allow then i will try again and just run the black command.

**Comment by djeck1432** [comment]
1. REad documentation how to use `black`
2. Apply only for files which you have changes, NOT For the whole project
3. @sigmadawg

**Comment by sigmadawg** [comment]
> @sigmadawg If you're having issues, I could help work on it and put up a PR in One hour, if that's okay with you.
> 
> cc @djeck1432

Yes, i'd like the help

**Pull request closed without merging**